diff --git a/node_modules/react-native-volume-control/android/build.gradle b/node_modules/react-native-volume-control/android/build.gradle
index 72c39ac..ac16875 100644
--- a/node_modules/react-native-volume-control/android/build.gradle
+++ b/node_modules/react-native-volume-control/android/build.gradle
@@ -7,7 +7,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.4.0'
     }
 }
 
diff --git a/node_modules/react-native-volume-control/android/build/generated/source/buildConfig/debug/com/rtmalone/volumecontrol/BuildConfig.java b/node_modules/react-native-volume-control/android/build/generated/source/buildConfig/debug/com/rtmalone/volumecontrol/BuildConfig.java
deleted file mode 100644
index 01476d8..0000000
--- a/node_modules/react-native-volume-control/android/build/generated/source/buildConfig/debug/com/rtmalone/volumecontrol/BuildConfig.java
+++ /dev/null
@@ -1,13 +0,0 @@
-/**
- * Automatically generated file. DO NOT MODIFY
- */
-package com.rtmalone.volumecontrol;
-
-public final class BuildConfig {
-  public static final boolean DEBUG = Boolean.parseBoolean("true");
-  public static final String APPLICATION_ID = "com.rtmalone.volumecontrol";
-  public static final String BUILD_TYPE = "debug";
-  public static final String FLAVOR = "";
-  public static final int VERSION_CODE = 1;
-  public static final String VERSION_NAME = "1.0";
-}
diff --git a/node_modules/react-native-volume-control/android/src/main/java/com/rtmalone/volumecontrol/RNVolumeControlModule.java b/node_modules/react-native-volume-control/android/src/main/java/com/rtmalone/volumecontrol/RNVolumeControlModule.java
index 341c92d..5376370 100644
--- a/node_modules/react-native-volume-control/android/src/main/java/com/rtmalone/volumecontrol/RNVolumeControlModule.java
+++ b/node_modules/react-native-volume-control/android/src/main/java/com/rtmalone/volumecontrol/RNVolumeControlModule.java
@@ -7,7 +7,7 @@ import android.content.BroadcastReceiver;
 import android.content.IntentFilter;
 import android.media.AudioManager;
 
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
 import android.util.Log;
 
 import com.facebook.react.bridge.ActivityEventListener;
